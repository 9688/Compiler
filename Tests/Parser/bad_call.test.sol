Ошибка в строке 1 позиции 10: требуется выражение
Ошибка в строке 2 позиции 15: отсутствует )
"ROOT"
