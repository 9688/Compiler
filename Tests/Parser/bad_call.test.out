Ошибка в строке 1 позиции 9: отсутствует )
Ошибка в строке 2 позиции 14: отсутствует )
"ROOT"
